source/functions/wecSimInitialize.m

```diff
@@ -273,40 +278,47 @@
     error(['Cannot run WEC-Sim with Morrison Element (simu.morisonElement) and "etaImport" wave type'])
 end
 
-% Check for morisonElement inputs for simu.morisonElement == 1
-if simu.morisonElement == 1
-    for ii = 1:length(body(1,:))
-        if body(ii).nhBody ~=1
-            %
-            [rgME,~] = size(body(ii).morisonElement.rgME);
-            %
-            for jj = 1:rgME
-                if true(isfinite(body(ii).morisonElement.z(jj,:))) == true
-                    warning(['"body.morisonElement.z" is not used for "simu.morisonElement = 1. Check body ',num2str(ii),' element ',num2str(jj)])
+% Check for morisonElement inputs for simu.morisonElement == 1 || simu.morisonElement == 2
+if length(simu.morisonElement) == 1 && simu.morisonElement == 0
+    warning(['"simu.morisonElement" needs to be a [n x 1] column vector where n is the number of bodies. WEC-Sim has assumed no bodies have active Morison Elements.'])
+    simu.morisonElement = zeros(length(body(1,:)),1);
```

Comment:
Is there a reason why we need to initialize the ME in the simulation class? With this revision it seems to me that we should just remove this propoerty from the simulation class, and allow users to define the ME for each body using e.g. `body(#).morisonElement.option`, where the default is `0`, but users can specify `body(#).morisonElement.option = 1` or `2`. Thoughts?

Reply:
@kmruehl If we move to the body class then this update will not be backwards compatible and my initial thought was to minimize this effect; however, per your pull request in #667, if we are planning to make other non backwards compatible changes then these should probably be all coordinated for a larger release and I can update to reflect these changes.

I'd also like to revisit the definitions between `body` and `simu` classes as if it is a simulation detail then perhaps we keep within `simu` rather than transfer to `body`. Because other simulations, such as nonlinear hydro or even the nonlinear yaw, should then also be updated to be in the `body` class to be consistent. 

We can use one of our next group meetings to discuss further.

Reply:
I agree, let's discuss during our next team meeting. I think we should target a major release that includes some of these revisions that are not backward compatible.